src/utils/create-parsers.js

```diff
@@ -0,0 +1,18 @@
+function createParsers(modules) {
```

Comment:
Can you add type definition for this function?

```
/**
 * @param {Array<[() => Promise<any>, string[]]>} modules
 * @returns {{[key: string]: () => Promise<any>}}
 */
```

Reply:
I don't think this is correct, we know modules[0][0] is function returns `import()`, so it's a promise resolves to a "module" with "default" and the default export contains `parsers`. The return type is also known. But I don't know how to do it.